package.json

```diff
@@ -37,7 +37,10 @@
     "ethers": "6.8.1",
     "ethers-multicall-provider": "^5.0.0",
     "lodash": "^4.17.21",
-    "long": "^5.2.3"
+    "long": "^5.2.3",
+    "@solana/web3.js": "1.95.3",
+    "@project-serum/anchor": "^0.26.0",
+    "bs58": "^6.0.0"
```

Comment:
Can't see a direct usage of bs58 in this pull request
is it something mandatory to have installed manually to be paired with web3js or anchor ?